matomo/templatetags/matomo_tags.py

```diff
@@ -0,0 +1,17 @@
+from django import template
+from django.conf import settings
+
+
+register = template.Library()
+
+
+@register.inclusion_tag('matomo_tracking_code.html', takes_context=True)
+def matomo_tracking_code(context):
+    site_id = settings.MATOMO_SITE_ID
+    url = settings.MATOMO_URL
+
+    context.update({
+        'site_id': site_id,
+        'url': url
+    })
+    return context
```

Comment:
As mentioned elsewhere, I think the `matomo_tracking` variable should be added here too, instead of in the processor, however, with a couple of changes:

- rename key from 'matomo_tracking' to 'tracking_enabled'
- the value should consider the other required settings (site ID and server URL) i.e. tracking is only enabled if all the settings agree

We should be more explicit in the name of the 'url' variable and rename it to 'matomo_server_url', to make it very clear which URL and which _server_ URL we are referring to.

I'd like to rename the templatetag (and template) to 'matomo_tracking_tags'.

A unit test could be written for this templatetag, to make sure that 'tracking_enabled' is set correctly in the context. Are there any edge cases that could be problematic? The code is quite simple though, so I won't insist on it.